src/Spectre.libWavelet/WaveletCoefficients.h

```diff
@@ -1,5 +1,6 @@
 /*
 * WaveletCoefficients.h
+* Decouple wavelet coefficients & constants from decomposer
```

Comment:
Interesting... :)

Reply:
How is it possible that the commit name has suddenly became a license comment ?

Reply:
:D

Reply:
Guess I'll just remove that and we can proceed with merging :smile: Good catch though, it took my ~7 reads until I realised that this sentence is completely unrelevant.